html/modules/custom/hr_paragraphs/hr_paragraphs.module

```diff
@@ -1516,6 +1516,10 @@ function hr_paragraphs_menu_local_actions_alter(&$local_actions) {
   if (isset($local_actions['group_content.group_node_add_page'])) {
     $local_actions['group_content.group_node_add_page']['title'] = t('Add new page');
   }
+
+  if (isset($local_actions['group_content.group_node_create_page'])) {
+    $local_actions['group_content.group_node_create_page']['title'] = t('Add new page');
+  }
```

Comment:
really had to stare at this to see how it's not an exact duplicate of the statement above it 😆